census/models/utils.js

```diff
@@ -133,13 +133,17 @@ var processEntries = function(data, options) {
     data.submitters = [];
 
     if (options.cascade) { data.entries = cascadeEntries(data.entries, options.year); }
+
+    _.each(data.entries, function(e) {
+      e.computedYCount = e.yCount(data.questions);
```

Comment:
@pwalsh We divide entries on several groups (pending, rejected and other). So If we need this property in all entries we should initialize this property before dividing.